src/translations/en.json

```
@@ -6827,6 +6827,10 @@
                 }
               }
             }
+          },
+          "picker": {
+            "title": "Select Z-Wave network",
+            "no_entries": "No Z-Wave networks configured. Please set up Z-Wave JS integration first."
```

nit: Maybe include "*the* Z-Wave [JS] integration", so this reads a bit better?
```suggestion
            "no_entries": "No Z-Wave networks configured. Please set up the Z-Wave JS integration first."
```